bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/AbstractLinkRegistry.java

```diff
@@ -23,7 +24,12 @@
  * @param <L>
  *            Concrete type of the abstract link
  */
```

Comment:
Although this PR has been merged a while ago, I just now stumbled across the extension and found that the second parameter has not been added to the javadoc. I personally feel uncomfortable with so many generics so it would be good for understanding to document all generic type parameters. Especially, since I still have some implementation of the removed ItemThingLinkRegistry, that is based on this class and must now be changed too.